packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -215,9 +242,9 @@ class DriveCommand extends RunCommandBase {
       if (error is ToolExit) {
         rethrow;
       }
-      throwToolExit('CAUGHT EXCEPTION: $error\n$stackTrace');
+      throw 'Unable to run test: $error\n$stackTrace';
     } finally {
-      driver?.quit();
+      await driver?.quit();
```

Comment:
Is this where the 15 second timeout happens?

Reply:
Not really. The test process will take 15 seconds before exiting.

Reply:
sg, thanks